corecfg/services_test.go

```diff
@@ -34,12 +34,16 @@ import (
 )
 
 type servicesSuite struct {
+	coreCfgSuite
+
 	systemctlArgs [][]string
 }
 
 var _ = Suite(&servicesSuite{})
 
 func (s *servicesSuite) SetUpSuite(c *C) {
+	s.coreCfgSuite.SetUpSuite(c)
+
 	systemd.SystemctlCmd = func(args ...string) ([]byte, error) {
```

Comment:
this looks strange, are we calling here  systemctl both directly and through the systemd pkg? or we don't need coreCfgSuite here?

Reply:
Indeed, excellent point. This mixing is a mistake that I will fix